src/test/scala/org/scalacheck/magnolia/MagnoliaTest.scala

```diff
@@ -94,71 +94,71 @@ object MagnoliaTest extends TestSuite {
           Gen.mapOf[Simple, Simple](arb) ==> arbMap
         }
 
-        'Set - {
+        Symbol("Set") - {
           val arbSet = Arbitrary.arbitrary[Set[Simple]]
 
           Gen.containerOf[Set, Simple](expectedSimpleGen) ==> arbSet
         }
       }
 
-      'ADTs - {
+      Symbol("ADTs") - {
```

Comment:
Wouldn't this work with strings instead of symbols?

Reply:
Probably, I just went with the suggestion from the warning but I'll try it with strings

Reply:
Thanks @kubukoz You're right on this. Would you mind approving?

Reply:
Sure